qemu/tests/cfg/driver_load_stress.cfg

```diff
@@ -0,0 +1,54 @@
+- driver_load_stress:
+    type = driver_in_use
+    start_vm = yes
+    kill_vm_on_error = yes
+    check_guest_bsod = yes
+    kill_vm_on_error = yes
+    login_timeout = 240
+    suppress_exception = False
+    sleep_time = 300
```

Comment:
@suqinhuang 
Could you help explain why so long for the sleep time? I have thought the cmd execution is hang.
sleep_time = 300 means the case needs 10mins at least and it's just for the one round.